WeatherDispatcher/src/main/java/com/example/WeatherDispatcher/model/WeatherResult.java

```diff
@@ -0,0 +1,25 @@
+package com.example.WeatherDispatcher.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+@Data
+public class WeatherResult {
+
+    private String name;
+    private Main main;
+    private Wind wind;
+
+
+    @Data
+    class Main {
+        @JsonProperty("temp")
```

Comment:
why to use @JsonProperty if the name is the same?

Reply:
done